packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -37,6 +32,20 @@ function getAttestationKey(attestationRequest: AttestationRequest): AttestationK
   }
 }
 
+// TODO proper localization
+function getSecurityCodeText(language: string | undefined) {
+  switch (language) {
+    case 'es':
+    case 'es-419':
+    case 'es-LA': {
+      return 'Código de seguridad de Celo'
```

Comment:
i'd recommend aligning with the strings for mobile i18n
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/i18n.ts#L86

so `es-US` and `es-LA`